arangod/RocksDBEngine/RocksDBEdgeIndex.cpp

```diff
@@ -647,6 +651,9 @@ class RocksDBEdgeIndexLookupIterator final : public IndexIterator {
     _totalCachedSizeEffective +=
         cache::kCachedValueHeaderSize + key.size() + size;
     ++_totalInserts;
+    if (isEmpty) {
+      ++_totalEmptyInserts;
+    }
```

Comment:
```suggestion
    _totalEmptyInserts += static_cast<uint64_t>(isEmpty);
```
[godbolt](https://godbolt.org/#g:!((g:!((g:!((h:codeEditor,i:(filename:'1',fontScale:14,fontUsePx:'0',j:2,lang:c%2B%2B,selection:(endColumn:2,endLineNumber:8,positionColumn:2,positionLineNumber:8,selectionStartColumn:2,selectionStartLineNumber:8,startColumn:2,startLineNumber:8),source:'%23include+%3Calgorithm%3E%0A%23include+%3Ccstdint%3E%0A%23include+%3Ccstring%3E%0A%0Avoid+foo1(bool+isEmpty,+uint64_t%26+_totalEmptyInserts)+%7B%0A++++if+(isEmpty)+%7B%0A++++++++%2B%2B_totalEmptyInserts%3B%0A++++%7D%0A%7D%0A%0Avoid+foo2(bool+isEmpty,+uint64_t%26+_totalEmptyInserts)+%7B%0A++++_totalEmptyInserts+%2B%3D+static_cast%3Cuint64_t%3E(isEmpty)%3B%0A%7D%0A%0Aint+main()+%7B%7D'),l:'5',n:'0',o:'C%2B%2B+source+%232',t:'0')),header:(),k:46.206174777603366,l:'4',m:100,n:'0',o:'',s:0,t:'0'),(g:!((h:compiler,i:(compiler:g131,deviceViewOpen:'1',filters:(b:'0',binary:'1',binaryObject:'1',commentOnly:'0',debugCalls:'1',demangle:'0',directives:'0',execute:'0',intel:'0',libraryCode:'0',trim:'1'),flagsViewOpen:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,libs:!(),options:'-O3+-std%3Dc%2B%2B20+-DNDEBUG+-std%3Dc%2B%2B20++-Wall+-fsized-deallocation+-mfxsr+-mmmx+-msse+-msse2+-mcx16+-msahf+-mpopcnt+-msse3+-msse4.1+-msse4.2+-mssse3+-mpclmul+-mavx+-mxsave+-maes',overrides:!(),selection:(endColumn:1,endLineNumber:1,positionColumn:1,positionLineNumber:1,selectionStartColumn:1,selectionStartLineNumber:1,startColumn:1,startLineNumber:1),source:2),l:'5',n:'0',o:'+x86-64+gcc+13.1+(Editor+%232)',t:'0')),header:(),k:39.49420079618994,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((h:output,i:(compilerName:'x86-64+gcc+13.1',editorid:2,fontScale:14,fontUsePx:'0',j:1,wrap:'1'),l:'5',n:'0',o:'Output+of+x86-64+gcc+13.1+(Compiler+%231)',t:'0')),k:14.299624426206703,l:'4',n:'0',o:'',s:0,t:'0')),l:'2',n:'0',o:'',t:'0')),version:4)

Reply:
Thanks for the suggestion. Added.